source/blog/2018-03-08-update-kernel-arguments-on-atomic-host.html.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Update Kernel arguments on Atomic Host
+author: rubao
+date: 2018-03-08 17:00:00 UTC
+published: true
+comments: false
+tags:
+- atomic
+- rpm-ostree
+---
+
+Users or adminstrators may want to change [kernel arguments](https://www.kernel.org/doc/html/v4.14/admin-guide/kernel-parameters.html) of Atomic Host for various reasons.
+Previously, it was hard for the users due to many of the steps involved,
+and the harmful consequeces that can occur if users accidentally make a mistake
+in the changing process.
+
+In this post, I want to introduce a command(`rpm-ostree ex kargs`) that
+allows users to change kernel arguments on Atomic Host. This command simplifies
+the process of changing kernel arguments. This command also lies
+beneath [rpm-ostree](https://github.com/projectatomic/rpm-ostree),
+and because of that, it benefits from many of the cool features from rpm-ostree.
+One of them is `rpm-ostree rollback`, which can allow users to undo their old changes
+they do not want.
+
+Note: This command is still experimental, so if you have seen any
+unexpected behavior happening, please report an issue to rpm-ostree. This
```

Comment:
I would link 'please report an issue' to https://github.com/projectatomic/rpm-ostree/issues/new as a convenience